packages/sdk-core/src/modules/breadcrumbs/BreadcrumbsManager.ts

```diff
@@ -34,14 +45,23 @@ export class BreadcrumbsManager implements BacktraceBreadcrumbs, BacktraceModule
      */
     private _enabled = false;
 
+    private readonly _limits: BreadcrumbLimits;
     private readonly _eventSubscribers: BreadcrumbsEventSubscriber[] = [new ConsoleEventSubscriber()];
     private readonly _interceptor?: (breadcrumb: RawBreadcrumb) => RawBreadcrumb | undefined;
     private _storage: BreadcrumbsStorage;
 
     constructor(configuration?: BacktraceBreadcrumbsSettings, options?: BreadcrumbsSetup) {
+        this._limits = {
```

Comment:
The problem is right now available in the existing Storage path. Making it deprecated won't solve the problem because you will see a lot of inconsistent behavior between storage set by the storage option and storage set by the storage factory. 

Maybe we should instead put this responsibility to setup defaults on the builder side. I think its a better idea than storing it here, because: 
- all default will be always applied to every implementation of the specific module
- I think it should be expected that the user provides those options and the behavior stays consistent across different implementations. 


What do you think about that. With this approach:
- we don't need the factory method - we don't need to deprecate anything on our side,
- it seems better to have it there rather than depending on the breadcrumbmanager.

Reply:
We discuss about this on the call and we want to stay with the current approach. The current library design forces us to use this approach